support-frontend/assets/pages/contributions-landing/components/ContributionThankYou/ContributionThankYouUsElectionNewsletter.jsx

```diff
@@ -0,0 +1,95 @@
+// @flow
+// $FlowIgnore - required for hooks
+import React, { useState } from 'react';
+import { css } from '@emotion/core';
+import { space } from '@guardian/src-foundations';
+import { Button } from '@guardian/src-button';
+import { SvgArrowRightStraight } from '@guardian/src-icons';
+import ActionContainer from './components/ActionContainer';
+import ActionHeader from './components/ActionHeader';
+import ActionBody from './components/ActionBody';
+import SvgNotification from './components/SvgNotification';
+import { OPHAN_COMPONENT_ID_US_ELECTION_NEWSLETTER } from './utils/ophan';
+import { trackComponentClick } from 'helpers/tracking/behaviour';
+
+const buttonContainer = css`
+  margin-top: ${space[6]}px;
+`;
+
+const signUpToNewsLetter = (email: string) => {
+  const formData = new FormData();
+  formData.append('email', email);
+  formData.append('listName', 'us-morning-newsletter');
+  formData.append('name', '');
+
+  fetch('https://www.theguardian.com/email', {
+    method: 'POST',
+    body: new URLSearchParams(formData),
```

Comment:
is the `URLSearchParams` necessary here?

Reply:
It seems to be, I'm not 100% sure why that's the case though. It was surprisingly difficult to find good examples of using fetch to post form data - it's just all json I guess

Reply:
I got it from [here](https://stackoverflow.com/a/46640266/11995118)